fdbserver/fdbserver.actor.cpp

```diff
@@ -508,9 +508,10 @@ void skipListTest();
 Future<Void> startSystemMonitor(std::string dataFolder,
```

Comment:
a nit: seems like the input to the routine are getting lengthier, maybe one potential approach is to create an `struct` could be to follow bellow pattern to abstract out params. 

```
struct SystemMonitorContext {
std::string dataFolder;
Optional<Standalone<StringRef>> dcId,
Optional<Standalone<StringRef>> zoneId,
Optional<Standalone<StringRef>> machineId,
Optional<Standalone<StringRef>> datahallId
}
```